src/utils/useExtension.ts

```diff
@@ -34,20 +34,14 @@ export const useExtension = () => {
   useEffect(() => {
     async function doEffect() {
       if (extensions.length) {
-        const allAccounts = await web3Accounts()
-        // TODO: We want to filter the account for the ones usable with the connected chain
         const genesisHash = await getGenesis()
+        // TODO: we can also subscribe using web3AccountsSubscribe, which would update the accounts list upon creation of a new account
+        const allAccounts = await web3Accounts({ genesisHash, ss58Format: 38 })
```

Comment:
ss58Format = 38 can be imported from the SDK by the way

Reply:
I did remember we had something like that, but not what it was called. The stakeboard does not use the sdk though, only type-definitions and augmentation.

Reply:
It’s conveniently called `ss58Format` :D

Reply:
Seeing a couple of @kiltprotocol dependencies I didn’t realize those are so minimal